vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -120,7 +120,7 @@ vic_cesm_run(vic_clock *vclock)
     // if save:
     if (vclock->state_flag) {
         log_warn("Skipping state file write");
-        // vic_store();
+        vic_store(&dmy_current);
```

Comment:
I think after deleting the unused variables, it should be `vic_store()` now. (But not sure why this didn't give an error?)